src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let discount = 0;
+
+  if (days >= 3) {
```

Comment:
Don't hardcode values, use a variable here and in other similar cases (checklist #1)